### PR TITLE
Refactors Applier function signature to accept scope

### PR DESCRIPTION
**Pull request by bshumpe**

### CODE REVIEW

File: go.mod
Comment (jt0) [inline go.mod:1]: NP: Can you remove this file from the PR?

File: structs/applierprovider.go [hunk 1]
```diff
@@ -68,7 +68,7 @@ var (
 // one can use the wildcard scope (e.g. "*:this_colon_:_does_not_indicate_a_scope").
 //
 // NB: scopes can't be reused within the input. If a scope repeats, the last one wins. This is true for wildcards
-//     (implicit, explicit, or both) as well.
+// (implicit, explicit, or both) as well.
```
Comment (jt0) [inline structs/applierprovider.go:71]: NP: please leave the extra spaces.

**Review by jt0 — CHANGES_REQUESTED** [review]
Two minor nitpicks...

**Comment by bshumpe** [comment]
Addressed nits.

**Pull request merged**